code/modules/mob/dead/observer/observer.dm

```diff
@@ -87,6 +88,8 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 
 	//starts ghosts off with all HUDs.
 	toggle_medHUD()
+
+	START_PROCESSING(SSobj, src)
```

Comment:
This starts the processing of all observers, regardless of `seerads` being true or not. I would move the toggles inside the verb.

Reply:
Fixed in 000b89a